driver/controller_test.go

```diff
@@ -181,14 +181,20 @@ func (*fakeTagsDriver) UntagResources(context.Context, string, *godo.UntagResour
 }
 
 func TestControllerExpandVolume(t *testing.T) {
+	default_volume := &godo.Volume{
+		ID:            "volume-id",
+		SizeGigaBytes: (defaultVolumeSizeInBytes / giB),
+	}
 	tcs := []struct {
-		name string
-		req  *csi.ControllerExpandVolumeRequest
-		resp *csi.ControllerExpandVolumeResponse
-		err  error
+		name        string
+		req         *csi.ControllerExpandVolumeRequest
+		resp        *csi.ControllerExpandVolumeResponse
+		err         error
+		godo_volume *godo.Volume
 	}{
 		{
-			name: "request exceeds maximum supported size",
+			name:        "request exceeds maximum supported size",
+			godo_volume: default_volume,
```

Comment:
Instead of requiring to define the volume for every current and future test case, could we make it optional and let the test body pick `defaultVolume` if the volume from the test case struct is nil?

Reply:
Sure, updated.